connlp/visualize.py

```diff
@@ -101,4 +106,25 @@ def network(self, docs, show=False, fig_width=10, fig_height=8, dpi=300, arrows=
                     fontsize=font_size)
 
         if show:
-            plt.show()
\ No newline at end of file
+            plt.show()
+            
+            
+            
+            
+    def wordcloud(self, data, stopword_list):
+    
+        self.data = data
+        self.stopword_list = stopword_list    
+ 
+        stopwords = set(STOPWORDS)
+        for word in self.stopword_list:
+            stopwords.add(word)
+        
+        wc = WordCloud(font_path='font/NanumGothic.ttf', background_color='white', stopwords=stopwords)
+        cloud = wc.generate(self.data)
+            
+        plt.figure(figsize=(22,22))
+        plt.imshow(cloud, interpolation='lanczos')
+        plt.axis('off') 
+        plt.show() 
+        plt.savefig(fname = 'wordcloud.png')
```

Comment:
Provide some customizing options with the figure (e.g., size, save or not, ...).